front-end/src/components/CustomAction/CustomDeleteItemsAction.js

```diff
@@ -1,39 +0,0 @@
-import * as _ from 'lodash'
```

Comment:
Quick check that this file and `VPLinkWidget` should be deleted? It looks like they weren't actually used and it doesn't look like their removal affects the functionality, but wanted to do a quick double-check.